script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -164,38 +175,39 @@ I walk up the stairs in the lobby.
 [sha:99b82c823d1d2b7594c0fa9ba008b9c02d00197d]{
 -- Page 363, Offset 15690.
 -- 豪華なシャンデリアを横目に楽しみながら、西館二階の端にある自室に足を向けた。
-I enjoy the sight of the gorgeous chandeliers out of the corner of my eyes as I head toward my room on the second floor of the west wing.
+I enjoy the sight of the gorgeous chandeliers out of the corner of my eye as I head toward my room on the second floor of the west wing.
 }
 [sha:cceb047af00da5deddd74d1f7eb1853592edbe38]{
 -- Page 364, Offset 15691.
 -- 「あ、お帰り～♪」
-"Oh, welcome home. ♪"
+"Oh, welcome hooome ♪"
 }
 [sha:eca3c59f10971d3ac97b50b35b67e6c95087d7da]{
 -- Page 364, Offset 15692.
 -- 「な／に／ぃ／！？」
 // weird formatting used in the Japanese here.
-"Wh-a-t?!"
+"Wh―a―t?!"
```

Comment:
not a fan of splitting mid syllable so I'd drop the first dash and just leave it "Wha―t?!"

Reply:
The VA is every explicitly three staccato syllables here, otherwise I'd agree.